ExaltedApotheosisPreReqs.cs

```diff
@@ -0,0 +1,59 @@
+    //cs_include Scripts/CoreBots.cs
+    //cs_include Scripts/CoreFarms.cs
+    using Skua.Core.Interfaces;
+
+    public class ExaltedApotheosisPreReqs
+    {
+        public IScriptInterface Bot => IScriptInterface.Instance;
+        public CoreBots Core => CoreBots.Instance;
+        public CoreFarms Farm = new();
+
+        public void ScriptMain(IScriptInterface bot)
+        {
+            Core.SetOptions();
+
+            PreReqs();
+
+            Core.SetOptions(false);
+        }
+
+        public void PreReqs()
+        {
+            Core.AddDrop("Exalted Node", "Thaumaturgus Alpha", "Apostate Alpha", "Exalted Relic Piece", "Exalted Forgemetal", "Exalted Artillery Shard");

```

Comment:
there is no need for as killMons method  already handels it internally and auto add them to drop list and unbank those
same for buy method